app/templates/verkiezingen/verkiezingsuitslag/prepare.hbs

```diff
@@ -40,13 +40,13 @@
     @showStartEndDate={{true}}
   />
 
-  <AuContent @skin="small">
+  <div class="au-o-box no-pagination au-u-padding-top-none">
```

Comment:
This no-pagination shouldn't be here, this is css for tables, don't think this has any effect here.